PasswordVerifierTest/PasswordVerifier_VerifyPasswordShould.cs

```diff
@@ -0,0 +1,66 @@
+using PasswordVerifier;
+using Xunit;
+
+namespace PasswordVerifierTest
+{
+    public class PasswordVerifier_VerifyPasswordShould
+    {
+        [Fact]
+        public void VerifyPassword_InputIsqwe123_ThrowsExceptionNoMinimum9Characters()
```

Comment:
do not use variable (password) value in method(test) name, this makes an impression that only for this specific value it will fail.